setup.cfg

```diff
@@ -1,4 +0,0 @@
-[bdist_wheel]
```

Comment:
Will this be done by default?

Reply:
This is just an empty `ini`-file section, no `universal = 1` there, which was only needed to mark the _pure python package_ as compatible to both Python 2 and Python 3. As Python 2 is end-of-life since a long time as `pyelftools` is Python 3.9+ only, it can be removed safely